spec.bs

```diff
@@ -2832,6 +2868,20 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
        navigable=], or the result of [=checking if unloading is user-canceled=] for |navigable|'s
        [=navigable/active document=]'s [=Document/inclusive descendant navigables=] otherwise.
 
+  Add a step after step 22.8.2 (Let |finalSandboxFlags| be the [=set/union=]...) that reads:
+
+  3. If |sourceSnapshotParams|' [=source snapshot params/target fenced frame config=] is not null,
```

Comment:
```suggestion
  3. If |sourceSnapshotParams|'s [=source snapshot params/target fenced frame config=] is not null,
```

I believe in standards we generally prefer the trailing `s` in these cases, even though both are grammatically correct. Can you fix this and the many instances of this below / elsewhere in this PR?

Reply:
Fixed 4 instances of `.*s|'` instead of `.*s|'s` in the spec file.